Oware/src/Harvest.java

```diff
@@ -0,0 +1,50 @@
+class Harvest
+{
+	public void harvest(Pit pitToEmpty, Player player)
+	{
+		if (player.getWhichPlayer())
```

Comment:
If the player knew its own store house, pits and opponents pits, the player could harvest its own pits, I think a more robust Player object would make a lot of this easier, but this is a clever minimalist solution